<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ScannedBarCode_2">
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Scan Barcode Here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="welomelineEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Welcome tro ATE Application</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="NewtestpushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>New Test</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reportpushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Generate Test Report</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLineEdit" name="barcodeLineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="currentLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Volt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="voltLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Touch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="touchComboBox">
           <property name="currentText">
            <string/>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="placeholderText">
            <string>Please choose touch test result</string>
           </property>
           <item>
            <property name="text">
             <string>Pass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fail</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="Finish">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Finished</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="reportBarCodelineEdit">
           <property name="placeholderText">
            <string>Scanned Bar Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generatepushButton">
           <property name="text">
            <string>GenerateReportPushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
